inference-engine/tests/functional/inference_engine/transformations/mul_add_conversion_test.cpp

```diff
@@ -233,10 +233,7 @@ INSTANTIATE_TEST_CASE_P(AddToPower, MulOrAddConversionTests, testing::Combine(
 
 
 INSTANTIATE_TEST_CASE_P(MulAddNegative, MulAddConversionTests, testing::Combine(
-        testing::Values(std::make_tuple(InputShape{DYN, 3, 64},
```

Comment:
Why you changed this test case?

Reply:
ScaleShift can be not only 4D. ScaleShift can be 2D, 3D, 4D, 5D. As result after fix, this test is not correct and was removed.